modules/Cargo.toml

```diff
@@ -36,6 +37,7 @@ dyn-clonable = "0.9.0"
 regex = "1"
 subtle-encoding = "0.5"
 sha2 = { version = "0.9.3", optional = true }
+flex-error = "0.4.0"
```

Comment:
Should we propagate the `std` feature to `flex-error`?

Reply:
Done.